addons/web/static/src/core/record_selectors/record_selector.js

```diff
@@ -22,6 +22,13 @@ export class RecordSelector extends Component {
         onWillUpdateProps((nextProps) => this.computeDerivedParams(nextProps));
     }
 
+    get isAvatarModel() {
+        // bof
```

Comment:
bof

Reply:
I would have expected a better message than this